prov/efa/src/rxr/rxr_msg.c

```diff
@@ -252,8 +249,8 @@ ssize_t rxr_msg_post_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
 
 	ctrl_type = delivery_complete_requested ? RXR_DC_LONG_MSGRTM_PKT : RXR_LONG_MSGRTM_PKT;
 	tx_entry->rxr_flags |= RXR_LONGCTS_PROTOCOL;
-	return rxr_pkt_post_ctrl_or_queue(rxr_ep, RXR_TX_ENTRY, tx_entry,
```

Comment:
Should this be changed in the emulated rdma paths as well? I'm seeing that this is called in `rxr_rma_post_efa_emulated_read` and `rxr_rma_post_write`. We probably can return EAGAIN for the atomic ops too.

Reply:
I think they should be. Eventhough using ```rxr_ep_post_ctrl_or_queue``` for RMA and atomics is not wrong, because we do not supported ordering for RMA and atomics. Still it is unnecessary, so I prefer to use ```rxr_ep_post_ctrl``` for them.

I have changed that in new revision.